content/en/about/board/2024-02-29.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+show_meta: false
+title: '2024-02-29'
+---
+
+### Date and Time of Meeting
+
+* Call to order: 2024-02-29, 9:05 CET
+* Adjourned: 9:53 CET
+* Next Meeting: 2024-03-28, 14:09 CET
+
+### Roll call
+
+#### Directors and Officers Present
+
+* Isabel Drost-Fromm (Chair)
+* Daniel Izquierdo-Cortazar (President)
+* Sebastian Spier (Assistant Secretary, Vice President)
+* Matt Cobby
+* Yuki Hattori
+
+#### Directors and Officers Absent
+
+* Katie Schueths
+* Dmitrii Sugrobov
+* Russel Rutledge (Executive Director)
+* Tom Sadler (Secretary)
+* Georg Gruetter (Treasurer)
```

Comment:
Not sure  I was the _Assistant_ Treasurer at the time of this meeting, yet. In any case, Tom and I took over the _Treasurer_ and _Assistant Treasurer_ role at the same time.

@tsadler1988 : Do you remember when we took over from Max and Silona?

Reply:
Yes we were in role by then, we took over sometime in between the 2023-11 and 2023-12 board meetings - https://github.com/InnerSourceCommons/foundation/blob/master/board/meeting/agenda-2023-12.md#secretary - it should've been in December's terse notes but those seem to be missing.